features/step_definitions/purap/requisition.rb

```diff
@@ -1,6 +1,23 @@
-And /^I create a Requisition with required Delivery and Additional Institutional information populated$/ do
+And /^I create a Requisition with required Chart-Organization, Delivery and Additional Institutional information populated$/ do
   visit(MainPage).requisition
   on RequisitionPage do |req_page|
+    # Check Chart/Org associated with the user; if either one does not exist, use corresponding parameter default value and perform lookup
+    chart,org = req_page.chart_org_readonly.split('/')
+    (chart.nil? || chart.empty?) ? (nil) : (chart = chart.strip())
+    (org.nil? || org.empty?) ? (nil) : (org = org.strip())
+    if ((org.nil? || org.empty?) || (chart.nil? || chart.empty?))
+      req_page.chart_org_search
+      on OrganizationLookupPage do |org_lookup|
+        (chart.nil? || chart.empty?) ? (chart_for_lookup = get_aft_parameter_value(ParameterConstants::DEFAULT_CHART_CODE)) : (chart_for_lookup = chart)
```

Comment:
What I was getting at with this one is that we want to do the assignment from the result of the ternary instead of within it. I.e. I think this should be reformulated to:
```ruby
chart_for_lookup = (chart.nil? || chart.empty?) ? get_aft_parameter_value(...) : chart
```
Then you'd do something similar for `org`. You'd also update the condition based on what we settle on regarding finding `chart`/`org` above.

Alternatively, since you don't use `chart`/`org` after this point, you could simply change this to:

```ruby
chart =  get_aft_parameter_value(...) if (chart.nil? || chart.empty?) # changing condition as above
org =  get_aft_parameter_value(...) if (org.nil? || org.empty?) # changing condition as above
```
and simply use those variables wherever you have `chart_for_lookup`/`org_for_lookup` now.

I think this would be valuable mostly in that it would be more readable.